src/modules/localStorage.js

```diff
@@ -0,0 +1,106 @@
+const entryTask = document.getElementById('newTask');
+
+/* eslint-disable */
+class Task {
+  constructor(description, completed = false, index) {
+    this.description = description,
+    this.completed = completed,
+    this.index = index
+  }
+}
```

Comment:
[OPTIONAL] You could simplify your code importing and exporting functions without using classes. You are using it in some parts of your code but not in the entire code.